resource/packages/apt/apt.go

```diff
@@ -0,0 +1,88 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apt
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/asteris-llc/converge/resource"
+)
+
+// Package is an API for package state
+type Package struct {
+	resource.TaskStatus
+
+	Name  string
+	State string
```

Comment:
I suggest making State a type: `type State string` then having constants for the different states. You can look at user.user or user.group if you want to see this done elsewhere.